app/create-app-template.js

```diff
@@ -104,11 +108,15 @@ module.exports = api => {
       )
       .join('\n')}
 
+    if (entry.extendRootOptions) {
+      entry.extendRootOptions(rootOptions)
+    }
+
```

Comment:
why after `enhanceAppFiles` 🤔

Reply:
I was thinking that **user** provided entry callback should be called last (to have chance to correct all previous plugin-imposed setup). This change is not needed for the PR. If you dislike it I can revert.